doc/backend_vars.asciidoc

```diff
@@ -135,12 +135,16 @@ QEMU_QMP_CONNECT_ATTEMPTS;integer;20;The number of attempts to connect to qemu q
 PATHCNT;integer;2;Number of paths in MULTIPATH scenario
 PXEBOOT;boolean or 'once';0;Boot VM from network, on every boot or only once if set to 'once'
 QEMU;QEMU binary filename;undef;Filename of QEMU binary to use
+QEMUCLUSTERS;integer;0;Number of CPU clusters used by VM
+QEMUCORES;integer;0;Number of CPU cores used by VM
```

Comment:
Is 0 really the default? I guess the default is just an null-string reverting to whatever qemu defaults to which is likely "1" in this case, isn't it?

Reply:
Changed all 5 default values to `undef`.